src/components/layout/layout-overlay.tsx

```diff
@@ -8,10 +8,6 @@ import { bannerTypes } from 'common/constants'
 const LiveChat = Loadable(() => import('./livechat'))
 const WhatsApp = Loadable(() => import('./whatsapp'))
 
-type TProps = {
```

Comment:
remove is_ppc as it was unused , before we had CFD warning and ppc was used there, which now we apply it to the homepage overlay (which is the new atomic design overlay ) .